modules/rosa-additional-principals-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-accessing-cluster.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="rosa-additional-principals-overview_{context}"]
+= Additional principals on your ROSA cluster
+
+You can approve additional principals to connect to your cluster. You can add the following principal types:
```

Comment:
"You can allow AWS IAM Roles as additional principals to connect to your cluster's private API Server endpoint."